site/layouts/shortcodes/js-dismiss.html

```diff
@@ -5,11 +5,15 @@
 Dismissal can be achieved with the `data` attribute on a button **within the {{ $name }}** as demonstrated below:
 
 ```html
-<button type="button" class="btn-close" data-bs-dismiss="{{ $name }}"><span class="visually-hidden">Close</span></button>
+<button type="button" class="btn-close" data-bs-dismiss="{{ $name }}" data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-title="Close">
+  <span class="visually-hidden">Close</span>
+</button>
 ```
 
 or on a button **outside the {{ $name }}** using the `data-bs-target` as demonstrated below:
 
 ```html
-<button type="button" class="btn-close" data-bs-dismiss="{{ $name }}" data-bs-target="#my-{{ $name }}"><span class="visually-hidden">Close</span></button>
+<button type="button" class="btn-close" data-bs-dismiss="{{ $name }}" data-bs-target="#my-{{ $name }}" data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-title="Close">
+  <span class="visually-hidden">Close</span>
+</button>
```

Comment:
Not sure tooltip should be added on triggers 2nd example (see [http://https://deploy-preview-1823--boosted.netlify.app/docs/5.2/components/alerts/#triggers](http://https//deploy-preview-1823--boosted.netlify.app/docs/5.2/components/alerts/#triggers)). Should the button in this case have a btn-close class? (same in Alerts, Toats, Offcanvas and Modals)